src/lib.rs

```diff
@@ -290,6 +308,19 @@ impl<M> Context<M> {
     {
         self.system_handle.subscribe_recipient::<M, E>(self.myself.clone());
     }
+
+    /// Subscribe current actor to event of type `E` and send the last cached event to it.
+    /// This is part of the event system. You don't need to call this method to receive
+    /// direct messages sent using [`Addr`] and [`Recipient`].
+    ///
+    /// Note that subscribing twice to the same event would result in duplicate events -- no
+    /// de-duplication of subscriptions is performed.
+    pub fn subscribe_and_receive_latest<E: Event + Into<M>>(&self) -> Result<(), SendError>
```

Comment:
Clarifying the docstring as per live discussion with @goodhoko.

```suggestion
    /// Subscribe current actor to event of type `E` and send the last cached event to it.
    /// This is part of the event system. You don't need to call this method to receive
    /// direct messages sent using [`Addr`] and [`Recipient`].
    ///
    /// Note that subscribing twice to the same event would result in duplicate events -- no
    /// de-duplication of subscriptions is performed.
    ///
    /// This method may fail if it is not possible to send the latest event. In this case it is
    /// guaranteed that the subscription did not take place. You can safely try again.
    pub fn subscribe_and_receive_latest<E: Event + Into<M>>(&self) -> Result<(), SendError>
```